piccolo/columns/base.py

```diff
@@ -594,7 +594,7 @@ def ilike(self, value: str) -> Where:
         For SQLite, it's just proxied to a LIKE query instead.
 
         """
-        if self._meta.engine_type == "postgres":
+        if self._meta.engine_type == "postgres" or self._meta.engine_type == "cockroach":
```

Comment:
We can do this instead - just slightly more compact:

```python
if self._meta.engine_type in ('postgres', 'cockroach'):
```

Reply:
Figured the other would be faster but I like the readability of `if .. in ()` :+1: